packages/financial-templates-lib/src/logger/Logger.js

```diff
@@ -34,6 +34,7 @@ const { transports } = require("./Transports");
 // By calling `await waitForLogger(Logger)`, with the local Logger instance, the process will wait for all upstream
 // transports to clear. This enables slower transports like slack to still send their messages before the process yields.
 async function waitForLogger(logger) {
+  if (require("minimist")(process.argv.slice(), {})._.indexOf("test") != -1) return true; // Exit early if in a test.
```

Comment:
`logger.end();` breaks the Winston logger within tests & basically means that you can't re-log from the same logger during the test. To fix this you have two options: 1) re-create the logger each time (we do this in the before each) or 2) don't end the logger instance if in a test. I'm actually happy to re-factor the tests such that they make a new logger each time so that we don't need this.

Reply:
After playing around with this I've managed to re-factor the tests to pass without this. I added an 
```
afterEach(async function() {
    serverInstance.close();
  });
```

block that will close the server. I moved the initialization of the logger and the server to the `beforeEach` block.

Reply:
Oh, gotcha, that makes sense. Didn't think about the fact that we're using the same logger across multiple tests. Either way of dealing with this is cool with me!